sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/DirectoryAPITest.groovy

```diff
@@ -3737,6 +3738,41 @@ class DirectoryAPITest extends APISpec {
         null     | null       | null        | null         | garbageLeaseID
     }
 
+    @Ignore("Requires manual OAuth setup and creates 5000+ files")
```

Comment:
Were you able to manually run this test to validate this feature?

Reply:
Yup, was able to successfully test this and on existing delete tests to ensure existing behavior remains the same.